pkgs/applications/networking/browsers/chromium/browser.nix

```diff
@@ -7,14 +7,28 @@ mkChromiumDerivation (base: rec {
   packageName = "chromium";
   buildTargets = [ "mksnapshot" "chrome_sandbox" "chrome" ];
 
+  # Trying to run NaCl apps with a stripped nacl_irt_*.nexe file results in a
+  # segfault. We therefore disable stripping here and manually strip the
+  # binaries when they're copied.
+  dontStrip = true;
```

Comment:
As mentioned before, we *really* should build NaCl as a separate derivation, that's why we have the `mkChromiumDerivation`. The NaCl derivation could then be referenced either via the wrapper or from the browser derivation, although I'd prefer the latter because not everybody will enjoy recompiling the main browser *and* NaCl just for the sake to have NaCl support.